### PR TITLE
Fix #2932 (Fix CEF hardware acceleration not working)

### LINKED ISSUE #2932: CEF Vulkan errors
### Describe the bug

So, when I launch CEF browser in MTA, there are Vulkan errors in cefdebug.txt. WebGL is not working, probably because of that(but who knows).

obraz

### Steps to reproduce

1. start resource `webbrowser`
2. open browser
3. check `cefdebug.txt`

### Version

v1.5.9-release-21622

### Additional context

Tested on
```
System: Windows 11
GPU: AMD R9 290X
GPU Driver version: 21.09.13.02-220802a-382934C-RadeonSoftware (latest for my gpu)
```
and
```
System: Windows 10
GPU: Intel HD 4400
```

### Relevant log output

```shell
[0324/164955.320:ERROR:angle_platform_impl.cc(43)] RendererVk.cpp:138 (VerifyExtensionsPresent): Extension not supported: VK_KHR_surface
[0324/164955.320:ERROR:angle_platform_impl.cc(43)] RendererVk.cpp:138 (VerifyExtensionsPresent): Extension not supported: VK_KHR_win32_surface
[0324/164955.320:ERROR:angle_platform_impl.cc(43)] Display.cpp:1021 (initialize): ANGLE Display::initialize error 0: Internal Vulkan error (-7): A requested extension is not supported, in ..\..\third_party\angle\src\libANGLE\renderer\vulkan\RendererVk.cpp, initialize:1574.
[0324/164955.320:ERROR:gl_display.cc(508)] EGL Driver message (Critical) eglInitialize: Internal Vulkan error (-7): A requested extension is not supported, in ..\..\third_party\angle\src\libANGLE\renderer\vulkan\RendererVk.cpp, initialize:1574.
[0324/164955.320:ERROR:gl_display.cc(932)] eglInitialize SwANGLE failed with error EGL_NOT_INITIALIZED
[0324/164955.320:ERROR:gl_initializer_win.cc(133)] GLDisplayEGL::Initialize failed.
[0324/164955.321:ERROR:viz_main_impl.cc(186)] Exiting GPU process due to errors during initialization
[0324/164955.380:ERROR:gpu_init.cc(523)] Passthrough is not supported, GL is disabled, ANGLE is
```

### Security Policy

- [X] I have read and understood the Security Policy and this issue is not security related.

### PR DESCRIPTION
**Pull request by TFP-dev**
Fixes #2932 and fixes WebGL not working.

Current MTA:
https://youtu.be/Mca4FgfQqMo

Fixed:
https://youtu.be/tepQ-iWNKIM

No issues found, browser works as smooth as it was before this fix (but WebGL is now working)

### CODE REVIEW

**Comment by patrikjuvonen** [comment]
It has been disabled for security reasons.

**Comment by TFP-dev** [comment]
> It has been disabled for security reasons.

Why? RAGE Multiplayer have it enabled in their CEF and I can't see any "security reasons" why it should be disabled in MTA.

**Comment by AlexTMjugador** [comment]
@Lpsd mentioned some other reasons why GPU acceleration is disabled on MTA's CEF implementation on Discord:

Discord message

**Comment by Pirulax** [comment]
Okay, so merging this would be nice.
We *just* have to investigate the abovementioned "security reasons" associated with it.

File: Client/cefweb/CWebApp.cpp [hunk 1]
```diff
@@ -28,7 +28,10 @@ void CWebApp::OnRegisterCustomSchemes(CefRawPtr<CefSchemeRegistrar> registrar)
 void CWebApp::OnBeforeCommandLineProcessing(const CefString& process_type, CefRefPtr<CefCommandLine> command_line)
 {
     command_line->AppendSwitch("disable-gpu-compositing");
-    command_line->AppendSwitch("disable-gpu");
+    
+    // "disable-gpu" disables hardware acceleration and then WebGL is not working
+    // command_line->AppendSwitch("disable-gpu");
+
```
Comment (TheNormalnij) [inline Client/cefweb/CWebApp.cpp]:
```suggestion
```

File: Client/cefweb/CWebApp.cpp [hunk 2]
```diff
@@ -28,7 +28,10 @@ void CWebApp::OnRegisterCustomSchemes(CefRawPtr<CefSchemeRegistrar> registrar)
 void CWebApp::OnBeforeCommandLineProcessing(const CefString& process_type, CefRefPtr<CefCommandLine> command_line)
 {
     command_line->AppendSwitch("disable-gpu-compositing");
```
Comment (Lpsd) [inline Client/cefweb/CWebApp.cpp]: Should also remove this and test to make sure it still works okay.

**Review by Lpsd — CHANGES_REQUESTED** [review]
Looks good, we discussed this in #ask-developers on Discord and believe the security issues are no longer a concern.

Worth removing the `disable-gpu-compositing` flag too as this will improve performance.

All changes in this PR should be excluded from the maetro release (W7/8).

File: Client/cefweb/CWebApp.cpp (hunk 2 above)
Comment (Pirulax) [reply Client/cefweb/CWebApp.cpp]:
```suggestion
```

Comment (Pirulax) [reply Client/cefweb/CWebApp.cpp]: Let first the other CI finish so we have artifacts to test with!

**Review by Lpsd — APPROVED** [review]

**Pull request merged**